numpy/matrixlib/defmatrix.py

```diff
@@ -1024,8 +1024,8 @@ def _from_string(str, gdict, ldict):
             except KeyError:
                 try:
                     thismat = gdict[col]
-                except KeyError:
-                    raise KeyError("%s not found" % (col,))
+                except KeyError as e:
+                    raise NameError("name {!r} is not defined".format(col)) from None
```

Comment:
```suggestion
                    raise NameError(f"name {col!r} is not defined") from None
```
Changing to an f-string brings this under the 80-char limit. If you want to stick with the original `.format` style that's fine, but please modify so that it's less than 80-char per line.